server/cert-s/openssl.conf

```diff
@@ -0,0 +1,94 @@
+[ req ]
+default_bits        = 2048
+default_keyfile     = server-key.pem
+distinguished_name  = subject
+req_extensions      = req_ext
+x509_extensions     = x509_ext
+string_mask         = utf8only
```

Comment:
Is this file meant to be in here?

Reply:
oops. no, will pull it out. It's a handy conf for generating self-signed certs 🙄